connect/client.go

```diff
@@ -404,7 +431,45 @@ func (rs *restClient) GetFileContent(file *onepassword.File) ([]byte, error) {
 	if content, err := file.Content(); err == nil {
 		return content, nil
 	}
+	response, err := rs.retrieveDocumentContent(file)
+	content, err := readResponseBody(response, http.StatusOK)
+	if err != nil {
+		return nil, err
+	}
+	file.SetContent(content)
+	return content, nil
+}
+
+func (rs *restClient) DownloadFile(fileUUID, itemUUID, vaultUUID, targetDirectory string) (string, error) {
+	file, err := rs.GetFile(fileUUID, itemUUID, vaultUUID)
+	if err != nil {
+		return "", err
+	}
+	response, err := rs.retrieveDocumentContent(file)
+	if err != nil {
+		return "", err
+	}
+
+	path := filepath.Join(targetDirectory, filepath.Base(file.Name))
+	dst, err := os.Create(path)
```

Comment:
I am only now realizing that this truncates existing files (i.e. existing files are overwritten). If this is what we want, we should definitely document this clearly. We could also consider not truncating existing files by using `os.OpenFile()` instead. What do you think?

Reply:
Nice spot!

I implemented a potential fix: this allows for the user to set an `overwrite` flag, if they wish to overwrite, in case a file already exists under the same path. In case this parameter is set to `false`, a file will get created in case it does not exist, and otherwise the error `fmt.Errorf("a file already exists under the %s path. you may want to enable the `overwrite` flag", path)` is thrown. 

We may want to rename this parameter to `force`, as it is usually named in commands, or maybe we'd like to transform the overwrite parameter to be variadic (case in which we'd check for only one bool value to be passed within the function), mimicking, in this way, having an optional parameter.

Let me know what you think about this.

Reply:
The reason for using a generic "force" command in the command line is to make it as widely applicable as possible. In this specific case, the argument does exactly one thing. So I think you're doing the user a great service with the current name. We could maybe even make it a bit more self-explanatory by calling it `overwriteExisting` or `overwriteIfExists`.

Reply:
Changed the name to `overwriteIfExists`!